fundamentals/installation.md

```diff
@@ -45,14 +45,37 @@ If you installed napari with `pip` you can upgrade by calling
 $ pip install napari --upgrade
 ```
 
-## choosing a different backend
-By default, napari uses [PySide2](https://wiki.qt.io/Qt_for_Python). This is what most users will want. 
+## choosing different Qt backends
 
-If you want to use [PyQt](https://www.riverbankcomputing.com/software/pyqt/intro) instead for your backend, you can run these lines after installing napari:
+Most users can safely ignore this section. If, however, you aim to use napari together with software that uses PyQt5, or have some other reason to prefer that Qt backend, read on.
+
+[Qt](https://www.qt.io) is a C++ library that provides unified graphical user interface (GUI) elements and interactivity across platforms and operating systems. Several libraries exist for accessing these elements from Python, which we call in the context of napari a *backend*.
+
+The default napari backend is [PySide2](https://wiki.qt.io/Qt_for_Python), and this is what you get if you type `pip install napari` or `conda install -c conda-forge napari` and you do not have [pyqt5](https://pypi.org/project/PyQt5/) installed. If you wish to use [PyQt5](https://www.riverbankcomputing.com/software/pyqt/intro) as the backend instead, you can install pyqt5 and napari from PyPI like this.
```

Comment:
I would refer to `PySide2` and `PyQt5` as different python bindings for our Qt backend. Isn't that more accurate. I might also introduce [qtpy](https://github.com/spyder-ide/qtpy) as an abstraction layer that allows us to use either set of python bindings